src/main/java/com/oltpbenchmark/api/collectors/monitoring/README.md

```diff
@@ -0,0 +1,19 @@
+# Monitoring in BenchBase
+
+Monitoring in BenchBase can be enabled using the
+```text
+ -im,--interval-monitor <arg>   Monitoring Interval in milliseconds
+ -mt,--monitor-type <arg>       Type of Monitoring (throughput/advanced)
+```
+command line option when executing BenchBase, where the monitoring interval describes the sleeping period of the thread between recording monitoring information.
+We currently support two types of monitoring: 1) Basic throughput monitoring to track the progress while executing a benchmark (mt=throughput) and 2) monitoring of query and system properties via system tables for both SQLServer and Postgres (mt=advanced).
```

Comment:
```suggestion
We currently support two types of monitoring:

1. Basic throughput monitoring to track the progress while executing a benchmark (`-mt=throughput`), and
2. monitoring of query and system properties via system tables for both SQLServer and Postgres (`-mt=advanced`).
    Support for other engines can also be added.

```

Might also be worth editing this in vscode with the markdown lint check extension.